bootstrap/config/config.go

```diff
@@ -550,3 +555,19 @@ func (cp *Processor) listenForChanges(serviceConfig interfaces.Configuration, co
 func (cp *Processor) logConfigInfo(message string, overrideCount int) {
 	cp.lc.Infof("%s (%d envVars overrides applied)", message, overrideCount)
 }
+
+func getMapDiff(m1 config.InsecureSecrets, m2 config.InsecureSecrets) []string {
+	var updatedPaths []string
```

Comment:
Also needs to handle newly add Paths and probably Paths that got removed/changed